javascript/javascript1/week3/Js1week3.js

```diff
@@ -0,0 +1,189 @@
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+const indexNameToRemove = names.indexOf("Ahmad");
+names.splice(indexNameToRemove, 1);
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'] */
+
+//When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function timeToDestination(speed, destinationDistance) {
+    arrivingTime = destinationDistance / speed;
+    diff = arrivingTime - Math.floor(arrivingTime);
+    if (diff == 0) {
+        return console.log(`You will arrive in ${arrivingTime} hours`);
+    }
+
+    else
+        return console.log(`You will arrive in ${Math.floor(arrivingTime)} hours and ${Math.round(diff * 60)} minutes`);
+}
+
+
+timeToDestination(50, 432); // 8 hours and 38 minutes
+
+
+//Series duration of my life
+
+const seriesDurations = [
+    {
+        title: "friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "Breaking Bad",
+        days: 6,
+        hours: 0,
+        minutes: 50,
+    },
+    {
+        title: "Modern family",
+        days: 10,
+        hours: 9,
+        minutes: 0,
+    },
+];
+expectedAge = 80 * 365;
+function percentageOfAge(array) {
+    let sum = 0;
+    for (i = 0; i < array.length; i++) {
+        const totalDayFormat = [];
+        const percentage = [];
+        totalDayFormat[i] = (array[i].days + (array[i].hours / (24)) + (array[i].minutes / (24 * 60)));
+        percentage[i] = (totalDayFormat[i] / expectedAge) * 100;
+        result = console.log(`${array[i].title} took ${percentage[i].toFixed(3)}% of my life.`);
+        sum += totalDayFormat[i];
+        totalPercentage = (sum / expectedAge) * 100;
+
+    }
+    console.log(`In total that is ${totalPercentage.toFixed(3)} of my life`);
+    return result;
+}
+percentageOfAge(seriesDurations)
+
```

Comment:
Great work! ✨
Please remember to declare your variables with either let or const on line 63 and 73.
From my perspective, you should not save your console.log statement in the variable result. Simply have line 71 be
console.log(`${array[i].title} took ${percentage[i].toFixed(3)}% of my life.`);

Additionally, you can omit the return statement on line 77, since your console log statement is happening for each index of the array in your for loop as is. 😊